### PR TITLE
fix : event zoom clone

### PR DESCRIPTION
**Pull request by hdz-666**

### CODE REVIEW

**Comment by triptu** [comment]
let's remove the older images too

File: src/views/components/gridView.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const GridCenterView = ({
       ) : EVENT_ROOM_IDS.some(id => window.location.href.includes(id)) ? (
         <div className="h-full w-full grid place-items-center p-5">
           <a
-            href="https://community.100ms.live/facetime-clone"
+            href="https://community.100ms.live/meeting"
```
Comment (triptu) [inline src/views/components/gridView.jsx]: url not found

Comment (hdz-666) [reply src/views/components/gridView.jsx]: ohh ok my mistake i copied wrong url

Comment (hdz-666) [reply src/views/components/gridView.jsx]: i will remove other images too

**Review by triptu — APPROVED** [review]

**Pull request merged**